package.json

```diff
@@ -36,15 +50,15 @@
     "node": "16.20.0"
   },
   "scripts": {
-    "vscode:prepublish": "npm run compile",
     "compile": "tsc -p ./",
     "watch": "tsc -watch -p ./",
     "pretest": "npm run compile && npm run lint",
     "lint": "eslint src --ext ts",
     "test": "node ./out/test/runTest.js",
     "test-coverage": "node ./out/test/runTest.js --coverage",
     "prettier:write": "prettier --write \"src/**/*.{ts, js}\"",
-    "prettier:verify": "prettier --list-different \"src/**/*.{ts, js}\""
+    "prettier:verify": "prettier --list-different \"src/**/*.{ts, js}\"",
+    "vscode:prepublish": "npm run pretest && npm run test && npm run compile"
```

Comment:
`pretest` already does the compilation so no need to to do another compilation at the end

Reply:
No sure why I saw lint run twice though. I must be hallucinating. Still a good change though!